spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -1193,6 +1194,11 @@ private boolean determineSslEnabled(boolean sslEnabled) {
 		 */
 		private String password;
 
+		/**
+		 * Name of the stream.
+		 */
+		private String name;
```

Comment:
I wonder if creating a `stream` namespace wouldn't be better. At the very least it should be `streamName` but perhaps we'd get more strem-related properties in the future?

Reply:
Yes, I think we should add a `stream` namespace.

Reply:
Sorry, I missed that reviewing the change. It's all good as it is.